App crash, "cannot read property 'addr' of null"

## Summary

I saw the following crash earlier today, 2024-05-12:
<img width=200 src="https://github.com/daimo-eth/daimo/assets/169280/128ec7ce-e574-4bda-8c19-f744d9ca663d">

Persists on quitting and restarting the app.

## Investigation

Debug log shows following:

```
2024-05-12T06:58:12.135Z log [APP] rendering dcposch, onboarded
2024-05-12T06:58:12.136Z log [ERROR] rendering Cannot read property 'addr' of null TypeError: Cannot read property 'addr' of null
    at anonymous (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:334024:19)
    at map (native)
    at InviteesBubbles (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:334021:37)
    at renderWithHooks (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:28364:24)
    at beginWork$1 (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:30929:34)
    at performUnitOfWork (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:30626:27)
    at workLoopSync (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:30620:55)
    at renderRootSync (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:30606:19)
    at performSyncWorkOnRoot (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:30381:36)
    at flushSyncCallbacks (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:27599:33)
    at batchedUpdatesImpl (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:31318:174)
    at batchedUpdates (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:26903:32)
    at _receiveRootNodeIDEvent (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:26925:19)
    at receiveEvent (/Users/expo/Library/Developer/Xcode/DerivedData/Daimo-fqyuburyhitsbcesjtzisutllqsr/Build/Intermediates.noindex/ArchiveIntermediates/Daimo/BuildProductsPath/Release-iphoneos/main.jsbundle:26944:30)
    at apply (native)
```

--- a/packages/daimo-api/src/api/getAccountHistory.ts
+++ b/packages/daimo-api/src/api/getAccountHistory.ts
@@ -169,7 +169,9 @@ export async function getAccountHistory(
     : null;
 
   const inviteeAddrs = inviteGraph.getInvitees(address);
-  const invitees = inviteeAddrs.map((addr) => nameReg.getDaimoAccount(addr)!);
+  const invitees = inviteeAddrs
+    .map((addr) => nameReg.getDaimoAccount(addr))
+    .filter((acc) => acc != null) as EAccount[];
 
   // Get pfps from linked accounts
   const profilePicture = profileCache.getProfilePicture(address);

